doc/source/tutorial/part08/02_emle.rst

```diff
@@ -0,0 +1,435 @@
+
+>>> from emle.calculator import EMLECalculator
+>>> calculator = EMLECalculator(device="cpu")
+
+By default, ``emle-engine`` will use `TorchANI <https://aiqm.github.io/torchani/>`_
+as the backend for in vacuo calculation of energies and gradients. However,
+it is possible to use a wide variety of other backends, including your own
+as long as  it supports the standand `Atomic Simulation Environment (ASE) <https://wiki.fysik.dtu.dk/ase/>`_
+`calculator <https://wiki.fysik.dtu.dk/ase/ase/calculators/calculators.html>`_ interface.
+For details, see the `backends <https://github.com/chemle/emle-engine#backends>`_
+section of the ``emle-engine`` documentation. At present, the default embedding
+model provided with ``emle-engine`` supports only the elements H, C, N, O, and S.
+We plan on adding support for other elements in the near future.
+
+Creating a QM engine
+--------------------
+
+We now need to set up the molecular system for the QM/MM simulation and create
+an engine to perform the calculation:
+
+>>> qm_mols, engine = sr.qm.emle(
+...     mols,
+...     mols[0],
+...     calculator,
+...     cutoff="7.5A",
+...     neighbour_list_frequency=20
+... )
+
+Here the first argument is the molecules that we are simulating, the second
+selection coresponding to the QM region (here this is the first molecule), and
+the third is calculator that was created above. The fourth and fifth arguments
+are optional, and specify the QM cutoff distance and the neighbour list update
+frequency respectively. (Shown are the default values.) The function returns a
+modified version of the molecules containing a "merged" dipeptide that can be
+interpolated between MM and QM levels of theory, along with an engine. The
+engine registers a Python callback that uses ``emle-engine`` to perform the QM
+calculation.
+
+Running a QM/MM simulation
+--------------------------
+
+Next we need to create a dynamics object to perform the simulation. For QM/MM
+simulations it is recommended to use a 1 femtosecond timestep and no constraints.
+In this example we will use the ``lambda_interpolate`` keyword to  interpolate
+the dipeptide potential between pure MM (λ=0) and QM (λ=1) over the course of
+the simulation, which can be used for end-state correction of binding free
+energy calculations.
+
+>>> d = qm_mols.dynamics(
+...     timestep="1fs",
+...     constraint="none",
+...     qm_engine=engine,
+...     lambda_interpolate=[0, 1],
+...     platform="cpu",
+... )
+
+We can now run the simulation. The options below specify the run time, the
+frequency at which trajectory frames are saved, and the frequency at which
+energies are recorded. The ``energy_frequency`` also specifies the frequency
+at which the λ value is updated.
+
+>>> d.run("1ps", frame_frequency="0.05ps", energy_frequency="0.05ps")
+
+.. note::
+
+    Updating λ requires the updating of force field parameters in the ``OpenMM``
```

Comment:
Do we still need to mention this now that the new version of OpenMM is out with the optimised updateParametersInContext?

Reply:
Good point, this was more for Exs' benefit so can be safely removed.